src/coreComponents/common/format/table/TableLayout.hpp

```diff
@@ -125,32 +456,125 @@ class TableLayout
   string_view getTitle() const;
 
   /**
-   * @return The border margin, number of spaces at both left and right table sides
+   * @param title The table title
+   * @return The tableLayout reference
+   */
+  TableLayout & setTitle( string_view title );
+
+  /**
+   * @brief Remove the return line at the end & begenning of the table
+   * @return The tableLayout reference
+   */
+  TableLayout & disableLineBreak();
```

Comment:
With factory patterns, you should do as such:
```suggestion
  TableLayout & addLineBreaks( bool enable );
```

Reply:
Still an issue: your new version `TableLayout & disableLineBreak( bool value );` doesn't have the right meaning I think: the internal variable is named `m_wrapLine`. Something is wrong here, I think `disableLineBreak()` should be renamed `enableLineBreak()`